### PR TITLE
NMS-8302: Use the same ObjectMapper as the JacksonJaxbJsonProvider

### PR DESCRIPTION
**Pull request by j-white**
JIRA: http://issues.opennms.org/browse/NMS-8302
Bamboo: http://bamboo.internal.opennms.com:8085/browse/OPENNMS-ONMS613

### CODE REVIEW

File: features/bsm/test-util/src/main/java/org/opennms/netmgt/bsm/test/BsmTestUtils.java [hunk 1]
```diff
@@ -188,8 +191,16 @@ public static ReductionKeyEdgeResponseDTO toResponseDTO(SingleReductionKeyEdgeEn
     // convert to json
     public static <T> String toJson(T input) {
         Objects.requireNonNull(input);
+
+        final ObjectMapper mapper = new ObjectMapper();
+        final AnnotationIntrospector introspectorPair = AnnotationIntrospector.pair(
+                new JacksonAnnotationIntrospector(),
+                new JaxbAnnotationIntrospector());
+        mapper.setDeserializationConfig(mapper.getDeserializationConfig().withAnnotationIntrospector(introspectorPair));
+        mapper.setSerializationConfig(mapper.getSerializationConfig().withAnnotationIntrospector(introspectorPair));
```
Comment (mvrueden) [inline features/bsm/test-util/src/main/java/org/opennms/netmgt/bsm/test/BsmTestUtils.java]: Should we add a `createDefaultObjectMapper()` method somewhere? Then we do not have to create the `ObjectMapper` manually all the time.

Comment (j-white) [inline features/bsm/test-util/src/main/java/org/opennms/netmgt/bsm/test/BsmTestUtils.java]: I had a feeling you were going to bring that up :)

**Comment by mvrueden** [comment]
I forgot to mention this yesterday, but can we also enable the disabled JSON tests on the BusinessServiceRestServiceIT?
See https://github.com/OpenNMS/opennms/blob/jira/NMS-8302/features/bsm/rest/impl/src/test/java/org/opennms/web/rest/v2/bsm/BusinessServiceRestServiceIT.java#L371 and https://github.com/OpenNMS/opennms/blob/jira/NMS-8302/features/bsm/rest/impl/src/test/java/org/opennms/web/rest/v2/bsm/BusinessServiceRestServiceIT.java#L540

**Comment by mvrueden** [comment]
Awesome :+1: :) :+1:

**Pull request merged**